### DISCUSSION TITLE
Page based router and Layout

### DISCUSSION THREAD

**Discussion by devmnj** [solidjs/solid#1086] [Ideas]
Everything good so far. It's 🚀.

What I miss was 

- Page base routing such one found in Nextjs
- Layout mechanism found on Nuxt

**Comment by maxmilton** [comment]
Have you seen <https://github.com/solidjs/solid-app-router>? Additionally, doing a search brings up a bunch of alternative options:
- https://github.com/topics/solidjs?q=router
- https://github.com/search?q=solid+router
